docs/benchmarks/rabi_oscillations.ipynb

```diff
@@ -0,0 +1,495 @@
+    "\n",
+    "print(\"Noisy state after measurement:\" + str(noisy_post_measurement_state))\n",
+    "print(\"Noisy state before measurement:\" + str(noisy_pre_measurement_state))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "2h6yoOl4Rmwt"
+   },
+   "source": [
+    "## 2. Parameterized Circuits and Sweeps\n",
+    "\n",
+    "Now that you have some of the basics end to end, you can create a parameterized circuit that rotates by an angle $\\theta$:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "n6h6yuyGM58s"
+   },
+   "outputs": [],
+   "source": [
+    "import sympy\n",
+    "\n",
+    "theta = sympy.Symbol(\"theta\")\n",
+    "\n",
+    "parameterized_circuit = cirq.Circuit(\n",
+    "    cirq.rx(theta).on(my_qubit), cirq.measure(my_qubit, key=\"out\")\n",
+    ")\n",
+    "SVGCircuit(parameterized_circuit)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "rU3BBOp0S4sM"
+   },
+   "source": [
+    "In the above block you saw that there is a `sympy.Symbol` that you placed in the circuit. Cirq supports symbolic computation involving circuits. What this means is that when you construct `cirq.Circuit` objects you can put placeholders in many of the classical control parameters of the circuit which you can fill with values later on.\n",
+    "\n",
+    "Now if you wanted to use `cirq.simulate` or `cirq.sample` with the parameterized circuit you would also need to specify a value for `theta`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "SMdz-yAZSwrU"
+   },
+   "outputs": [],
+   "source": [
+    "samples_at_theta_equals_2 = sim.sample(\n",
+    "    parameterized_circuit, params={theta: 2}, repetitions=10\n",
+    ")\n",
+    "samples_at_theta_equals_2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "H_H13Hc8g873"
+   },
+   "source": [
+    "You can also specify *multiple* values of `theta`, and get samples back for each value."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "0zjZxGY6hIsu"
+   },
+   "outputs": [],
+   "source": [
+    "samples_at_multiple_theta = sim.sample(\n",
+    "    parameterized_circuit, params=[{theta: 0.5}, {theta: 3.141}], repetitions=10\n",
+    ")\n",
+    "samples_at_multiple_theta"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "juuWvOEphaaE"
+   },
+   "source": [
+    "Cirq has shorthand notation you can use to sweep `theta` over a range of values."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "8lCb3049hqXn"
+   },
+   "outputs": [],
+   "source": [
+    "samples_at_swept_theta = sim.sample(\n",
+    "    parameterized_circuit,\n",
+    "    params=cirq.Linspace(theta, start=0, stop=3.14159, length=5),\n",
+    "    repetitions=5,\n",
+    ")\n",
+    "samples_at_swept_theta"
```

Comment:
In general, I don't think you need to assign to the variable unless you later use it.  Same with other blocks.
```
sim.sample(
     parameterized_circuit,
     params=cirq.Linspace(theta, start=0, stop=3.14159, length=5),
     repetitions=5,
)
```